src/service/translate.js

```diff
@@ -709,6 +710,29 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
 
   /**
    * @ngdoc function
+   * @name pascalprecht.translate.$translateProvider#useForceAsyncReload
+   * @methodOf pascalprecht.translate.$translateProvider
+   *
+   * @description
+   * If force async reload is enabled, async loader will be always called
+   * even if $translationTable already contains the language key.
+   *
```

Comment:
So then it means: The previous value is overridden with that one from the async response in case it was already set.